### PR TITLE
Add confeetti

### LINKED ISSUE #2: Confetti celebration animation

### PR DESCRIPTION
**Pull request by itscatherinelee**
Add party.js to shoot some confetti every time the mouse moves.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+            isConfettiOn.current = false;
+          });
+        }
+      };
+  
+      document.addEventListener('mousemove', shootConfetti);
+      return () => {
+        document.removeEventListener('mousemove', shootConfetti);
+      };
+    }, []);
```
Comment (syj67507) [inline src/App.tsx]: I recommend moving this to its own file.

Here is a reference for how you can do so from react docs.

```
// useConfetti.ts
export function useConfetti() {
  const isConfettiOn = useRef(false);
  useEffect(() => {
    // all your logic
  }, []);
}
```

**Review by syj67507 — APPROVED** [review]
Suggestion for moving the confetti stuff to its own hook, but not necessary.

**Comment by syj67507** [comment]
I also went ahead and linked it to the confetti issue on the issues tab.

File: src/App.tsx (hunk 1 above)
Comment (itscatherinelee) [reply src/App.tsx]: oh very interesting. i will indeed do this practice. thank you for your review

Comment (itscatherinelee) [reply src/App.tsx]: resolved

**Review by syj67507 — APPROVED** [review]

**Pull request merged**